.github/workflows/release.yml

```diff
@@ -10,19 +29,35 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           fetch-depth: 0
+          token: ${{ secrets.CICD_RELEASES_PAT }}
 
       - name: Configure Git
         run: |
-          git config user.name "$GITHUB_ACTOR"
-          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+          git config --global user.email "cicd@piiano.com"
+          git config --global user.name "Github Actions"
 
       - name: Install dependency repos
         run: |
           helm repo add bitnami https://charts.bitnami.com/bitnami
 
+      - name: Update Vault Version
+        run: |
+          ./bump_versions.sh ${{ inputs.vault_version }}  ${{ inputs.bump-package }}
+
+          git_status=$(git status -s)
+
+          if [ -n "$git_status" ]; then
+            echo "There are changes to commit"
+            git add .
```

Comment:
@nirhaas - I wanted to skip adding files one by one. Is this reasonable/working?